Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -463,7 +487,7 @@ public AndConstraint<JTokenAssertions> ContainSubtree(string subtree, string bec
         /// </code>
         public AndConstraint<JTokenAssertions> ContainSubtree(JToken subtree, string because = "", params object[] becauseArgs)
         {
-            return BeEquivalentTo(subtree, true, because, becauseArgs);
+            return BeEquivalentTo(subtree, true, null, because, becauseArgs);
```

Comment:
🔧 Same here with the `null`.

Reply:
Was there a reason not to use `config => config` here and above?